.github/workflows/test-asan.yml

```
@@ -39,6 +39,6 @@ jobs:
       - name: Environment Information
         run: npx envinfo
       - name: Build
-        run: make build-ci -j2 V=1
+        run: make build-ci -j$(nproc) V=1
```

I think if you don't speciify a number, it would achieve the same: https://www.gnu.org/software/make/manual/make.html#Parallel
```suggestion
        run: make build-ci -j V=1
```

---

No, that's for `test.py` but not for `make`. Don't do that with `make`. https://github.com/nodejs/node/pull/8287#issuecomment-242870999